manifests/init.pp

```diff
@@ -61,4 +65,6 @@
     type   => 'dword',
     data   => $enabled_data,
   }
+
+  create_resources(windows_firewall::exception, $exceptions)
```

Comment:
Can we iterate over the elements here instead of using `create_resources`?

```suggestion
 $exceptions.each |$exception, $attributes| {
   windows_firewall::exception { $exception:
     * => $attributes,
   }
 }
```